doc/api/util.md

```diff
@@ -1085,7 +1089,8 @@ const options = {
     short: 'f'
   },
   bar: {
-    type: 'string'
+    type: 'string',
+    required: true
```

Comment:
I suggest not including `required` in the example, as it is a less common configuration. The majority of programs will not use `required`, and there is some small potential for confusion between options expecting an argument, and required options.